<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>526</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>672</width>
    <height>526</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>672</width>
    <height>526</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="settingWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="paramTab">
      <attribute name="title">
       <string>Parametre Ayarları</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkDefaultParam">
         <property name="text">
          <string>Hazır Parametre Kullan</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkCustomParam">
         <property name="text">
          <string>Özel Parametre Kullan</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="paramBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Parametreler</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideRight</enum>
            </property>
            <widget class="QWidget" name="param1">
             <attribute name="title">
              <string>Parametre 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="checkP">
                <property name="text">
                 <string>[-p] block passive DPI</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkR">
                <property name="text">
                 <string>[-r] replace host with hoSt</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkS">
                <property name="text">
                 <string>[-s] remove space between host header and its value</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkM">
                <property name="text">
                 <string>[-m] mix Host header case (test.com -&gt; tEsT.cOm)</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QCheckBox" name="checkF">
                  <property name="text">
                   <string>[-f] set HTTP fragmentation to value:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinF">
                  <property name="maximum">
                   <number>1337</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QCheckBox" name="checkK">
                  <property name="text">
                   <string>[-k] enable HTTP persistent (keep-alive) fragmentation and set it to value:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinK">
                  <property name="maximum">
                   <number>1337</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="checkN">
                <property name="text">
                 <string>[-n] do not wait for first segment ACK when -k is enabled</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QCheckBox" name="checkE">
                  <property name="text">
                   <string>[-e] set HTTPS fragmentation to value:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinE">
                  <property name="maximum">
                   <number>1337</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="checkA">
                <property name="text">
                 <string>[-a] additional space between Method and Request-URI (enables -s, may break sites)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkW">
                <property name="text">
                 <string>[-w] try to find and parse HTTP traffic on all processed ports (not only on port 80)</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QCheckBox" name="checkPort">
                  <property name="text">
                   <string>[--port] additional TCP port to perform fragmentation on (and HTTP tricks with -w):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinPort">
                  <property name="maximum">
                   <number>65535</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QCheckBox" name="checkIpId">
                  <property name="text">
                   <string>[--ip-id] handle additional IP ID (decimal, drop redirects and TCP RSTs with this ID):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineIpId"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="param2">
             <attribute name="title">
              <string>Parametre 2</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QCheckBox" name="checkDnsAddr">
                  <property name="text">
                   <string>[--dns-addr] redirect UDP DNS requests to the supplied IP address (experimental):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineDnsAddr">
                  <property name="text">
                   <string>208.67.220.220</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QCheckBox" name="checkDnsPort">
                  <property name="text">
                   <string>[--dns-port] redirect UDP DNS requests to the supplied port (53 by default)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinDnsPort">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>65535</number>
                  </property>
                  <property name="value">
                   <number>1253</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QCheckBox" name="checkDnsv6Addr">
                  <property name="text">
                   <string>[--dnsv6-addr] redirect UDPv6 DNS requests to the supplied IPv6 address (experimental):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineDnsv6Addr">
                  <property name="text">
                   <string>2a02:6b8::feed:0ff</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QCheckBox" name="checkDnsv6Port">
                  <property name="text">
                   <string>[--dnsv6-port] redirect UDPv6 DNS requests to the supplied port (53 by default)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinDnsv6Port">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>65535</number>
                  </property>
                  <property name="value">
                   <number>1253</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="checkBlacklist">
                <property name="text">
                 <string>[--blacklist] perform HTTP tricks only to host names and subdomains from
                          supplied text file. This option can be supplied multiple times.</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="quickSetting">
             <attribute name="title">
              <string>Hizli Ayarlar</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="1" column="0">
               <widget class="QGroupBox" name="groupQuickSettings">
                <property name="title">
                 <string>Hazir ayarlar</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="radioQuick1">
                   <property name="text">
                    <string>[-1] -p -r -s -f 2 -k 2 -n -e 2 (most compatible mode, default)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="radioQuick3">
                   <property name="text">
                    <string>[-3] -p -r -s -e 40 (better speed for HTTP and HTTPS)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="radioQuick2">
                   <property name="text">
                    <string>[-2] -p -r -s -f 2 -k 2 -n -e 40 (better speed for HTTPS yet still compatible)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QRadioButton" name="radioQuick4">
                   <property name="text">
                    <string>[-4] -p -r -s (best speed)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkQuickSettings">
                <property name="text">
                 <string>Hizli ayarlari aktif et</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="runTab">
      <attribute name="title">
       <string>Çalışma Ayarları</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QCheckBox" name="scheduleBox">
         <property name="text">
          <string>Çalışma Saatleri Ayarla (Sonraki guncellemede gelecek.)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="trayBox">
         <property name="text">
          <string>Sistem Tepsisine Küçült</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="startupBox">
         <property name="text">
          <string>Başlangıçta Otomatik Çalıştır</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="scheduleSettings">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Çalışma Saatleri</string>
         </property>
         <widget class="QDateTimeEdit" name="startTime">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>90</y>
            <width>51</width>
            <height>22</height>
           </rect>
          </property>
          <property name="displayFormat">
           <string>HH:mm</string>
          </property>
         </widget>
         <widget class="QDateTimeEdit" name="endTime">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>100</y>
            <width>51</width>
            <height>22</height>
           </rect>
          </property>
          <property name="displayFormat">
           <string>HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="notificationBox">
         <property name="text">
          <string>Bildirimleri Kapat</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
